ecs.tf

```diff
@@ -100,12 +100,13 @@ resource "aws_lb" "waggledance" {
 }
 
 resource "aws_lb_target_group" "waggledance" {
-  count       = var.wd_instance_type == "ecs" && var.enable_autoscaling ? 1 : 0
-  port        = local.wd_port
-  protocol    = "TCP"
-  target_type = "ip"
-  vpc_id      = var.vpc_id
-  tags        = var.tags
+  count              = var.wd_instance_type == "ecs" && var.enable_autoscaling ? 1 : 0
+  port               = local.wd_port
+  protocol           = "TCP"
+  target_type        = "ip"
+  preserve_client_ip = true
```

Comment:
Since this is open source and maybe other users might use this, should not we put this as a variable and then we set it ourselves?

Reply:
It was false before by default right?

Reply:
this was a bug when we added autoscaling group for ECS deployment, we are just fixiing the bug.